common/beerocks/scripts/prplmesh_utils.sh.in

```diff
@@ -276,21 +269,22 @@ status_function() {
     pgrep -l ieee1905_transport
     pgrep -l local_bus
 
-    # check for operational status
-    LOGS_PATH=@BEEROCKS_LOG_FILES_PATH@
+    bridge_mac="$(ifconfig @BEEROCKS_BRIDGE_IFACE@ | grep -o -E '([[:xdigit:]]{1,2}:){5}[[:xdigit:]]{1,2}')"
+    bml_cmd="bml_get_device_operational_radios $bridge_mac"
+    dbg "Executing BML: $bml_cmd"
 
     error=0
```

Comment:
I don't like the use of global variables.
You could, for example, return a string from this function, and decide that an empty string means an error occurred.